x-pack/plugins/cross_cluster_replication/public/__jest__/client_integration/follower_indices_list.test.js

```diff
@@ -11,7 +11,7 @@
       Could not load worker ReferenceError: Worker is not defined
           at createWorker (/<path-to-repo>/node_modules/brace/index.js:17992:5)
  */
```

Comment:
I ran the tests locally and it doesn't seem that this comment is relevant anymore, so we can get rid of it as well